src/apps/views/App/App.tsx

```diff
@@ -0,0 +1,60 @@
+import { appMessages } from "@saleor/apps/messages";
+import NotFoundPage from "@saleor/components/NotFoundPage";
+import useNavigator from "@saleor/hooks/useNavigator";
+import useNotifier from "@saleor/hooks/useNotifier";
+import React from "react";
+import { useIntl } from "react-intl";
+import { useLocation } from "react-router";
+
+import AppPage from "../../components/AppPage";
+import { useAppDetails } from "../../queries";
+import {
+  appDetailsUrl,
+  appsListPath,
+  getAppCompleteUrlFromDashboardUrl
+} from "../../urls";
+
+interface AppProps {
+  id: string;
+}
+
+export const App: React.FC<AppProps> = ({ id }) => {
+  const location = useLocation();
+  const { data } = useAppDetails({
+    displayLoader: true,
+    variables: { id }
+  });
+
+  const appExists = data?.app !== null;
+
+  const navigate = useNavigator();
+  const notify = useNotifier();
+  const intl = useIntl();
+
+  if (!appExists) {
+    return <NotFoundPage onBack={() => navigate(appsListPath)} />;
+  }
+
+  const appCompleteUrl = getAppCompleteUrlFromDashboardUrl(
+    location.pathname,
+    data?.app.appUrl,
+    id
+  );
+
+  return (
+    <AppPage
```

Comment:
It looks like we use same AppPage component for both app itself and configuration, this means we have the same breadcrumb: "App Name / **Configuration**". Change it accordingly

Reply:
Breadcrumbs are set automatically by the app in `useSettingsBreadcrumbs`

Reply:
Still, it needs to be changed to display proper breadcrumb for app main view (not "configuration")

Reply:
But where should bbe changed? In app or in dashboard?

Reply:
It's the app that sets the breadcrumb? Can't really see what's the use case here 🤔

Reply:
App sets breadcrumb via the event
```
if (event.data.type === ExtensionMessageType.BREADCRUMB_SET) {
  setBreadcrumbs(event.data.breadcrumbs);
}
```